bigchaindb/db/utils.py

```diff
@@ -86,33 +85,37 @@ def create_votes_secondary_index(conn, dbname):
     # wait for rethinkdb to finish creating secondary indexes
     r.db(dbname).table('votes').index_wait().run(conn)
 
+def init_database():
+   conn = get_conn()
+   dbname = get_database_name()
+   print(dbname)
+   create_database(conn, dbname)
+
+   table_names = ['bigchain', 'backlog', 'votes']
+   for table_name in table_names:
+     create_table(conn, dbname, table_name)
+
+   create_bigchain_secondary_index(conn, dbname)
+   create_backlog_secondary_index(conn, dbname)
+   create_votes_secondary_index(conn, dbname)
+
```

Comment:
The indentation should be four spaces. See [PEP 8](https://www.python.org/dev/peps/pep-0008/#indentation).